core/core_bind.cpp

```diff
@@ -1988,6 +2091,7 @@ void Engine::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("unregister_script_language", "language"), &Engine::unregister_script_language);
 	ClassDB::bind_method(D_METHOD("get_script_language_count"), &Engine::get_script_language_count);
 	ClassDB::bind_method(D_METHOD("get_script_language", "index"), &Engine::get_script_language);
+	ClassDB::bind_method(D_METHOD("capture_script_backtraces", "include_variables"), &Engine::capture_script_backtraces);
```

Comment:
```suggestion
	ClassDB::bind_method(D_METHOD("capture_script_backtraces", "include_variables"), &Engine::capture_script_backtraces, DEFVAL(false));
```
How about a default value? And in the native declarations too.

Reply:
Yeah, I considered it, but I'm still not sure. I didn't feel that it was warranted necessarily. It's not a method people are going to be calling all over the place, and it seemed reasonable to me that you'd be forced to make a conscious choice if you want the variables or not.

Reply:
Looking at [past breaking changes](https://docs.godotengine.org/en/stable/tutorials/migrating/index.html) it seems like pretty much all of them do it by adding an optional parameter at the end, which I guess makes sense, so that argument didn't really hold up.

Thinking about this some more, it probably does make sense to have this default to `false`, so I'll go ahead and make the change.